.travis/install.sh

```diff
@@ -36,6 +36,9 @@ if [[ "$DARWIN" = true ]]; then
             curl -O https://bootstrap.pypa.io/get-pip.py
             sudo python get-pip.py
             ;;
+        bundle)
```

Comment:
@glyph - I may have misunderstood what you were going for in this comment. 

I've been using the brew python to create a standalone version of the application. Yes, this does make the bundle bigger, but, it completely insulates it from any system python changes that could be introduced. I'm basically advocating for bundling the necessary python interpreter code with mimic, instead of having mimic require that python exist somewhere in the user's path.